README.md

```diff
@@ -186,7 +186,30 @@ via configuration.
        - path: '/health'
          format: 'health'
    ```
+   **OR**
 
+   Use `set_export`:
```

Comment:
I suggest to move set_export usage  up and instead of `3. After role initialization`. metrics are not exposed without `set_export` call until metrics role initialised and clusterwide config applied. `set_export` call is not optional but must have in init.lua. Also I suggest documenting `4. To view metrics via API endpoints,` as an optional method of runtime configuration because it is not required in case when `set_export` is used.